modules/efk-logging-configuring-image-about.adoc

```diff
@@ -9,15 +9,14 @@ Each cluster logging component uses an image.
 
 By default, cluster logging uses the following images:
 
-* *ELASTICSEARCH_IMAGE* deploys Elasticsearch, by default `docker.io/openshift/origin-logging-elasticsearch5:latest`.
-* *FLUENTD_IMAGE* deploys Fluentd, by default `docker.io/openshift/origin-logging-fluentd:latest`.
-* *KIBANA_IMAGE* deploys Kibana, by default `docker.io/openshift/origin-logging-kibana5:latest`.
-* *CURATOR_IMAGE* deploys Curator, by default `docker.io/openshift/origin-logging-curator5:latest`.
-* *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}, by default `docker.io/openshift/oauth-proxy:latest`.
-* *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`.
+* *ELASTICSEARCH_IMAGE* deploys Elasticsearch, by default `quay.io/openshift/origin-logging-elasticsearch5:latest`.
+* *FLUENTD_IMAGE* deploys Fluentd, by default `quay.io/openshift/origin-logging-fluentd:latest`.
+* *KIBANA_IMAGE* deploys Kibana, by default `quay.io/openshift/origin-logging-kibana5:latest`.
+* *CURATOR_IMAGE* deploys Curator, by default `quay.io/openshift/origin-logging-curator5:latest`.
+* *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}, by default `quay.io/openshift/origin-oauth-proxy:latest`.
+//* *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`.
 * *WATCH_NAMESPACE* allows cluster logging to view all namespaces in the cluster, by default `openshift-logging`.
```

Comment:
This is the namespace that the operator will watch for CR events in. It should be set by the installation means (this is typically OLM). For OLM it uses an annotation that it will also add to the deployment which corresponds to the operator group's namespace it is configured for.